yarn-project/noir-contracts/src/contracts/non_native_token_contract/src/main.nr

```diff
@@ -45,7 +51,7 @@ contract NonNativeToken {
         let owner_balance = balances.at(owner.x);
         let note = Note::new(initial_supply, owner);
 
-        // Insert note and emit encrypted note preimage via oracle call
+        // Insert new note to a set of user notes and emit the newly created encrypted note preimage via oracle call.
         context = owner_balance.insert(context, note);
         assert(emit_encrypted_log(inputs.call_context.storage_contract_address, owner_balance.storage_slot, note.owner, note) == 0);
 
```

Comment:
Is it worth explaining why we do `context.finish(inputs)`?

Reply:
Addressed in [78fa35a](https://github.com/AztecProtocol/aztec-packages/pull/923/commits/78fa35af3db83aa6191578502449b9dd85e9236b)